ENIGMAsystem/SHELL/Universal_System/Object_Tiers/object.cpp

```diff
@@ -82,7 +83,7 @@ namespace enigma
 
     void objectdata_load()
     {
-        objectdata = new objectstruct*[object_idmax];
+        objectdata.resize(object_idmax);
         for (int i = 0; i < objectcount; i++)
             objectdata[objs[i].id] = &objs[i];
```

Comment:
Okay, this is weird. I thought you wrapped this in a function. Where's it being declared, in this context?

Reply:
If you are referring to `objectdata` then this is refactored into an `AssetArray` now and it is being declared on `line 40` of this file.